sdks/java/core/src/test/java/org/apache/beam/sdk/io/TextIOTest.java

```diff
@@ -404,6 +406,37 @@ public void testCompressedRead() throws Exception {
 
   @Test
   @Category(NeedsRunner.class)
```

Comment:
Yes. `NeedsRunner` isn't really descriptive enough for what we want. This test requires a capability of Local File System Access.

Alternatively, the test could be written to access the `tempLocation` via `IOChannelUtils`